develocity-gradle.yml

```diff
@@ -274,35 +366,77 @@ spec:
-                          if (isAtLeast(gePluginVersion, '2.1')) {
-                              logger.lifecycle("Setting captureTaskInputFiles: $geCaptureTaskInputFiles")
-                              if (isAtLeast(gePluginVersion, '3.7')) {
-                                  ext.buildScan.capture.taskInputFiles = geCaptureTaskInputFiles
+                          if (isAtLeast(develocityPluginVersion, '2.1')) {
+                              if (isAtLeast(develocityPluginVersion, '3.7')) {
+                                  ext.buildScan.capture.taskInputFiles = develocityCaptureFileFingerprints
                               } else {
-                                  ext.buildScan.captureTaskInputFiles = geCaptureTaskInputFiles
+                                  ext.buildScan.captureTaskInputFiles = develocityCaptureFileFingerprints
                               }
                           }
                       }
+  
+                      eachDevelocitySettingsExtension(settings, [DEVELOCITY_CONFIGURATION_CLASS]) { ext ->
+                          ext.buildScan.capture.fileFingerprints = develocityCaptureFileFingerprints
+                      }
                   }
 
-                  if (geUrl && geEnforceUrl) {
-                      extensionsWithPublicType(settings, GRADLE_ENTERPRISE_EXTENSION_CLASS).collect { settings[it.name] }.each { ext ->
-                          logger.lifecycle("Enforcing Gradle Enterprise: $geUrl, allowUntrustedServer: $geAllowUntrustedServer")
-                          ext.server = geUrl
-                          ext.allowUntrustedServer = geAllowUntrustedServer
+                  if (develocityUrl && develocityEnforceUrl) {
+                      logger.lifecycle("Enforcing Develocity: $develocityUrl, allowUntrustedServer: $develocityAllowUntrustedServer, captureFileFingerprints: $develocityCaptureFileFingerprints")
+                  }
+  
+                  eachDevelocitySettingsExtension(settings, [GRADLE_ENTERPRISE_EXTENSION_CLASS]) { ext ->
+                      if (develocityUrl && develocityEnforceUrl) {
+                          ext.server = develocityUrl
+                          ext.allowUntrustedServer = develocityAllowUntrustedServer
+                      }
+  
+                      if (buildScanTermsOfUseUrl && buildScanTermsOfUseAgree) {
+                          ext.buildScan.termsOfServiceUrl = buildScanTermsOfUseUrl
+                          ext.buildScan.termsOfServiceAgree = buildScanTermsOfUseAgree
+                      }
+  
+                      if (isAtLeast(develocityPluginVersion, '2.1')) {
+                          if (isAtLeast(develocityPluginVersion, '3.7')) {
+                              ext.buildScan.capture.taskInputFiles = develocityCaptureFileFingerprints
+                          } else {
+                              ext.buildScan.captureTaskInputFiles = develocityCaptureFileFingerprints
+                          }
                       }
```

Comment:
Not needed for the same reason

Reply:
Done in https://github.com/gradle/develocity-gitlab-templates/pull/25/commits/41f7a89ea28c0773d451d46ccce02701d6bf074b